website/docs/r/resourcemanager_lien.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in
+#     .github/CONTRIBUTING.md.
+#
+# ----------------------------------------------------------------------------
+layout: "google"
+page_title: "Google: google_resourcemanager_lien"
+sidebar_current: "docs-google-resourcemanager-lien"
+description: |-
+  A Lien represents an encumbrance on the actions that can be performed on a resource.
+---
+
+# google\_resourcemanager\_lien
+
+A Lien represents an encumbrance on the actions that can be performed on a resource.
+
+
+## Example Usage
+
+```hcl
+resource "random_id" "r" {
+  byte_length = 8
+}
+
+resource "google_project" "project" {
+  project_id = "project-${random_id.r.hex}"
+  name = "A very important project!"
+}
+
+resource "google_resourcemanager_lien" "lien" {
+  parent = "projects/${google_project.project.number}"
+  restrictions = ["resourcemanager.projects.delete"]
+  origin = "machine-readable-explanation"
+  reason = "This project is very important to me!"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+
+- - -
+
+* `reason` -
+  (Optional)
+  Concise user-visible strings indicating why an action cannot be performed
+  on a resource. Maximum lenth of 200 characters.
+* `origin` -
+  (Optional)
+  A stable, user-visible/meaningful string identifying the origin
+  of the Lien, intended to be inspected programmatically. Maximum length of
+  200 characters.
+* `parent` -
```

Comment:
I feel like at the very least, `parent` and `restrictions` should be Required, shouldn't they?